### ISSUE TITLE
Incorrect kernel size

### ISSUE THREAD

**Issue by tejus-gupta** [Blade6570/Dialation-Pytorch#2]
There is an error in CAN_network.py. On line 74, the kernel size should be 7.
` nn.Conv2d(512, 4096, kernel_size=3, stride=1, padding=0, bias=True, dilation=4), #fc6 layer `.

For reference, here is the network definition in fyu/dilation - [Link](https://github.com/fyu/dilation/blob/0f105742e8c2202af12feb54374781ba55c3e112/network.py#L111).

**Comment by Blade6570** [comment]
Hey,  thanks for pointing out. I have corrected the issue. However, I didn't notice any noticeable changes in the result (visually).
